File: General/FizzBuzz/fizzbuzz.rb
def fizzbuzz (limit)
  (0..limit).each do |number|
    if (number.remainder(15) == 0)
      puts 'fizzbuzz'
    elsif (number.remainder(5) == 0)
      puts 'buzz'
    elsif (number.remainder(3) == 0)
      puts 'fizz'
    else
      puts number
    end
  end
end

fizzbuzz(100)

File: General/FizzBuzz/fizzbuzz.js
function fizzbuzz (limit) {
  for (var number = 0; number <= limit; number++) {
    if (number % 15 === 0) {
      console.log('fizzbuzz')
    } else if (number % 5 === 0) {
      console.log('buzz')
    } else if (number % 3 === 0) {
      console.log('fizz')
    } else {
      console.log(number)
    }
  }
}

fizzbuzz(100)
